eogrow/core/area/base.py

```diff
@@ -38,6 +39,11 @@ class AreaSchema(BaseSchema):
     )
 
 
+class PatchListSchema(BaseSchema):
+    input_folder_key: str = Field(description="The storage manager key pointing to the folder containing the file.")
+    filename: str = Field(description="A JSON file containing a list of EOPatch names.", regex=r"^.+\.(json|JSON)$")
```

Comment:
if you want to get regex fancy, you can also use the case insensitive flag

```suggestion
    filename: str = Field(description="A JSON file containing a list of EOPatch names.", regex=r"^.+\.(?i:json)$")
```

Reply:
it does work, but as discussed privately, we'll keep the current one so that the regex is a bit easier to read